storage/innobase/include/row0log.h

```diff
@@ -255,6 +256,51 @@ row_log_estimate_work(
 	const dict_index_t*	index);
 #endif /* HAVE_PSI_STAGE_INTERFACE */
 
+/** Find out if temporary log files encrypted.
+@return true if temporary log file should be encrypted, false if not */
+MY_ATTRIBUTE((warn_unused_result))
+bool
+log_tmp_is_encrypted();
+
+/** Check the row log encryption is enabled or not.
+It will enable the row log encryption. */
+void
+log_tmp_enable_encryption_if_set();
+
+/** Encrypt a temporary file block.
+@param[in]	src		block to encrypt or decrypt
```

Comment:
`block to encrypt`

Reply:
fixed this comment and corresponding one in .cc file